lua/fishing_mod/cl_shop_menu.lua

```diff
@@ -5,7 +5,6 @@ local FishingMod_spritePlus, a = Material("sprites/key_12")
 FishingMod_spriteMinus:SetInt("$flags", 2097152)
 FishingMod_spritePlus:SetInt("$flags", 2097152)
 local black = Color(0, 0, 0, 144)
-local selected = Color(255, 255, 255, 255)

```

Comment:
If you're going to use `color_white` might as well localize it instead of global lookup, otherwise no reason to remove `selected`